conftest.py

```diff
@@ -11,15 +11,26 @@
 import subprocess
 import sys
 import time
+import json
+
 from collections.abc import Iterable
 
+import paho.mqtt.client as mqtt
+
 import pytest
 from riotctrl.ctrl import RIOTCtrl
 
 import testutils.github
 import testutils.pytest
 from testutils.iotlab import IoTLABExperiment, DEFAULT_SITE
 
+APP_ID = "11-lorawan"
+DEVICE_ID = "riot_lorawan_1"
+DEVICE_ID_ABP = "riot_lorawan_1_abp"
+DEVEUI = "009E40529364FBE6"
+APPEUI = "70B3D57ED003B26A"
+DEVADDR = "26011EB0"
```

Comment:
IMHO those should also come from outside the framework (either as parameter or environment variable) as they are dependent on the deployment scenario (one would need to have access to your TTN setup if they test locally e.g.).

Reply:
yes, you are right. I will :)

Reply:
done as well. I used this variables to use only one LoRaWAN device for both procedures (I used the NWKSKEY and APPSKEY generated by Over The Air Activation and the ABP test passed :)